scripts.js

```diff
@@ -0,0 +1,266 @@
+        $("#loading").hide();
+        $(".paper").show();
+        $("footer").show();
+    }, 1000);
+}
+
+/**
+ * Call is made when user clicks Home btn 
+ * on the top navigation
+ * reload the page
+ * users will the default page
+ */
+function resetToHome() {
+    location.reload();
+}
+
+/**
+ *  function is called when user click one 
+ *  of the joke option
+ *  hides items items displayed on the home page
+ *  identifies the joke the user clicked
+ * @param {event} returns the event triggered by
+ * the option user chose on the form
+ */
+function renderHomePage(event) {
+    hideHomeItems();
+    $(".pagination").hide()
+    $(".chosen-joke").remove();
+    $(".home-display-joke").empty();
+    $("footer").hide();
+    $("home-display-joke").hide();
+    if (event.target.id === "icndb") {
+        curJokeLink = "icndb";
+        icndbFetch();
+    } else if (event.target.id === "daddy") {
+        curJokeLink = "daddy";
+        daddyFetch();
+    } 
+}
+
+/**
+ * this function identifies the type of joke
+ * that user selected and make appropriate fetch call
+ */
+function checkJokeTypeForFetch() {
+    if (curJokeLink === "icndb") {
+        icndbFetch();
+    } else if (curJokeLink === "daddy") {
+        daddyFetch();
+    }
+}
+
+/**
+ * function is called when user clicks the page 
+ * on the pagination
+ * identifies the page that user clicks
+ * @param {event} triggered by the page number that user clicks
+ */
+function loadNextPage(event) {
+    $(".pagination").hide();
+    $("#loading").show();
+    $(".chosen-joke").remove();
+    curPage = parseInt(event.currentTarget.innerText);
+    $("footer").hide();
+    checkJokeTypeForFetch();
+}
+
+/**
+ * this function is called when user clicks the forward 
+ * btn of the pagination
+ * hides the currently displayed items 
+ * and display loading icons to alarm users
+ * @param {event} triggered when user click the forward btn
+ */
+function changeToNextPageNav(event) {
+    $("#loading").show();
+    $(".pagination").hide();
+    $(".page-btn").remove();
+    $(".chosen-joke").remove();
+    $("footer").hide();
+    firstNumOfPageNav += 3;
+     for (let i = 2; i >= 0; i--) {
+        let pageBtn = $("<li class='page-item page-btn' id='" + (firstNumOfPageNav + i) + "-page-btn'><a class='page-link' href='#'>" + (firstNumOfPageNav + i) +  "</a></li>");
+        $(pageBtn).insertAfter(".arrow-btn");
+        $(".page-btn").click(loadNextPage);
+    }     
+    curPage = firstNumOfPageNav;
+    checkJokeTypeForFetch();
+}
+
+
+/**
+ * this function is called when user clicks the backward
+ * btn of the pagination
+ * hides the currently displayed items 
+ * and generate the correct pagination  for upcoming display
+ * Display loading icons to alarm user that their request has been received
+ * @param {event} triggered when user click the backward btn
+ * this function doesn't respond on the first pagination display(pg 1, 2,3)
+ */
+function changeToPrevPageNav(event) {
```

Comment:
A better name could be `hanldeChangeToPrevPage`. Same goes for the for the function above ^